std_msgs/Header header

# CPU Temp
int8 temp_package
int8 temp_core_0
int8 temp_core_1
int8 temp_core_2
int8 temp_core_3
int8 temp_core_4
int8 temp_core_5
int8 temp_core_6
int8 temp_core_7
int8 temp_core_8
int8 temp_core_9

# CPU Percent
int8 cpu_percent
int8 percent_0
int8 percent_1
int8 percent_2
int8 percent_3
int8 percent_4
int8 percent_5
int8 percent_6
int8 percent_7
int8 percent_8
int8 percent_9
int8 percent_10
int8 percent_11
int8 percent_12
int8 percent_13
int8 percent_14
int8 percent_15
int8 percent_16
int8 percent_17
int8 percent_18
int8 percent_19
